apps/src/templates/MultipleSectionsAssigner.jsx

```diff
@@ -229,6 +246,28 @@ MultipleSectionsAssigner.propTypes = {
 };
 
 const styles = {
+  modalBackdrop: {
+    position: 'fixed',
+    top: 0,
+    right: 0,
+    bottom: 0,
+    left: 0,
+    backgroundColor: '#000',
+    opacity: 0.6,
+    zIndex: 1000,
+  },
+  modal: {
+    position: 'fixed',
+    top: '10%',
+    left: '50%',
+    zIndex: 1050,
+    width: 560,
+    marginLeft: -280,
```

Comment:
It looks like this achieves centering by being half of the width?  Is it possible to generalize the solution with something like `transform: translateX(-50%)`?

Reply:
Yes, I plan to modify styling and add a shadow when the component is extracted out, as then I'll use SCSS modules off the bat. Are you okay with me finalizing the styling in the next PR? I'll tag you for reviewing the styles of that one –– this PR is just for the keyboard and screenreader accessibility part.